firebase-firestore/src/test/java/com/google/firebase/firestore/util/MapperTest.java

```diff
@@ -899,8 +907,12 @@ public void setValue(String value) {
   }
 
   private static <T> T deserialize(String jsonString, Class<T> clazz) {
+    return deserialize(jsonString, clazz, null);
```

Comment:
optional nit: Whenever passing literals like null, true, false, etc. I like to comment them so you can tell what they represent without having to look up the method signature.  i.e.:

```
return deserialize(jsonString, clazz, /*docRef=*/null);
```

In this case, since the methods are next to each other, consider this very optional.

Reply:
Added.